ui/src/utils/index.tsx

```diff
@@ -41,3 +41,19 @@ export function getTimeAgoFromNow(baseDate: Date): string {
 
   return `${distance} ago`
 }
+
+/**
+ * Method to map data base status to table status
+ * @param status Data base status
+ * @returns Table status (RepSyncStateT)
+ */
+export function mapToRepoSyncStateT(status: string) {
+  switch (status) {
+    case 'DONE':
+      return 'succeeded'
+    case 'RUNNING':
+      return 'running'
+    default:
+      return 'failed'
+  }
+}
```

Comment:
```suggestion
}

```

add linebreak at EOF

Reply:
Same, we will improve it with ESLint rules.